learn/validator-trust-score.md

```diff
@@ -0,0 +1,286 @@
+
+$$
+\bar{R} = \frac{\sum_{i=0}^{m-1} \left( 1-a\frac{i}{m-1} \right) r_i}{\sum_{i=0}^{m-1} \left( 1-a\frac{i}{m-1} \right)}, \quad a = 0.5
+$$
+
+Being $a$, the parameter determining how much the observation of the oldest epoch is worth relative to the observation of the newest epoch.
+
+#### Adjusting for High-Reliability Expectations
+
+The previous formula provides a weighted moving average score for validators reliability in block production, where a score of $0.9$ indicates a significant downtime of 10%[*](#notice-0.9), highlighting recent performance and the need for improved consistency.
+
+To better reflect the high standards required, we will plot the value on a circle, with $c$ as the parameter defining the slope of the arc.
+
+$$
+R=-c+1-\sqrt{-\bar{R}^{2}+2c\bar{R}+\left(c-1\right)^{2}}
+$$
+
+$$
+\text{Center of circle at (c,-c+1), where }
+c=-0.16
+$$
+
+<figure>
+
+<iframe src="https://www.desmos.com/calculator/zqemsh7yay?embed" aspect-video frameborder="0" allowfullscreen></iframe>
+
+<figcaption>
+
+Graph of the reliability score adjustment. The x-axis represents the reliability score, and the y-axis represents the adjusted reliability score.
+
+</figcaption>
+
+</figure>
+
+What we achieve with this adjustment is to penalise more severely those validators that have a low reliability score and are unable to produce blocks when expected.
+
+<small text-12 id="notice-0.9">
+* Using 10% is only a heavy approximation. The value of $0.9 could represent 10% downtime, but also 20% or 5%, depending on when the downtime occurred. We say 10% to help the reader understand the scale of the score.
+</small>
+
+### Liveliness
+
+The Liveness factor measures how often a validator is online and selected to produce blocks. We want validators to be active because it ensures the network runs smoothly and securely. Validators that are frequently online and producing blocks receive a higher score, promoting consistent participation and reliability.
+
+#### Why Liveness
+
+If a validator is not active producing blocks, it could still have a high Size and Reliability score. This would be misleading because they are not contributing to the operation of the network. The Liveness factor ensures that only active validators that are actually selected to produce blocks receive a higher score. We want to penalize validators that are not selected to produce blocks due of being inactive, jailed, offline, etc.
+
+We use the term _liveness_ instead of _uptime_ because _uptime_ suggests precision, as in server contexts where the exact online time can be measured. In our context, there is no way to measure how long a validator has been online. We can only know when the validators have been active and producing blocks, but there is no way of knowing when they are active but not producing blocks, or when they are offline. In summary, to avoid confusion, we use _liveness_ to represent how often a validator is actively selected to produce blocks.
```

Comment:
What about this?

---

We use the term _liveness_ instead of _uptime_ because _uptime_ implies precise measurement, as in server contexts where you can measure online time. In our case, we can't measure how long a validator has been online. We can only see when validators are active and producing blocks. There's no way of telling when they're active but not producing blocks, or when they're offline.

To be clear, a validator can be active and offline at the same time. It could be offline and not producing blocks because it's offline, or it could be active but not producing blocks because it hasn't been selected in a certain period. This is why we use _liveness_ to show how often a validator is selected to produce blocks.